### DISCUSSION TITLE
#3 Third Sprint Activity & Sequence Diagrams, Angular

### DISCUSSION THREAD

**Discussion by AlexMi-Ha** [CUMGroup/AMOGUS#17] [Blog]
Howdy fellow AMOGUS enjoyers ✌🦑,

In this sprint we worked on our Use Case Requirements Specifications. 
We decided to formulate the Use Cases Login and Register:
>  [Login UCRS](https://github.com/CUMGroup/AMOGUS/blob/main/documents/requirements/Use-Case-Realization%20Specification/ucrs_login.md)
> [Register UCRS](https://github.com/CUMGroup/AMOGUS/blob/main/documents/requirements/Use-Case-Realization%20Specification/ucrs_register.md)

Additionally we worked on Activity Diagrams for out Use Cases Game Loop, Login and Register:
> [Login Use Case](https://github.com/CUMGroup/AMOGUS/blob/main/documents/requirements/usecases/accounts.login.usecase.md)
> [Register Use Case](https://github.com/CUMGroup/AMOGUS/blob/main/documents/requirements/usecases/accounts.register.usecase.md)
> [Game Loop Use Case](https://github.com/CUMGroup/AMOGUS/blob/main/documents/requirements/usecases/gameloop.usecase.md)

You can find more information about our diagrams and modelling on the on the respective linked pages above. 😁

Furthermore our team members started getting used to Angular. The three newbies in our team are doing the tutorial while our more expert crewmates are helping out and already testing out design ideas in angular.

All in all we conclude this sprint to be a successful one and we can gladly say that we are good in time! 
✨(☞ﾟヮﾟ)☞ ☜(ﾟヮﾟ☜)✨

Yours truly,
CUMGroup

**Comment by Luca-Hackl** [comment]
Dear AMOGUS team,

your diagrams are very nice! It's easy to understand and well written. It would be very nice if you would have used a software like figma to create your mockups :) Maybe you could take that into consideration for your next blog entry.

Kind regards,
RALF

**Reply by jas20202** [reply]
Hello there RALF,

thank you for your nice feedback ^^ Your idea with the mockups is very great and we already opened up a task for that in Jira which will be taken on as soon as a team member is free. 

Yours truly
CUMGruop (Project AMOGUS)

**Comment by DarkressX** [comment]
Hey CUMGroup,

i really like that you already have an idea for a mockup for every Use-Case.
Looking over the documents in your git I have to say, it's nice to see that you added a table of contents at the top of every document.
This makes reading and quickly navigating though it very smooth.
I noticed that in your sequence diagram there seems to be a small mistake with the UML standard.
For returns, dotted arrows should be used.

Besides this, I really like your ideas and concepts. Keep on keeping on.

Best wishes,

Common Team

**Reply by jas20202** [reply]
Hey there Commom Team,

thanks for the nice comment and the feedback. We will be fixing our sequence diagrams in sprint 4 which is running this week 😊
We are very glad you took your time to look over our documentation here on git and it's good to hear that the tables of content are useful indeed!

Yours truly 
CUMGruop (Project AMOGUS)

**Comment by jan-nie** [comment]
Dear AMOGUS team,

your Use-Case-Realization Specifications look clearly and well written. Also your activity diagrams look structured and easy to understand. We could follow your thoughts and are looking forward to the finished product. You did a great job!

We wish you continued success with AMOGUS.
Framgång

**Reply by jas20202** [reply]
Dear Framgång,

thank you very much for your nice comment ^^
We are glad to hear that our UCRS and activity diagrams are well to understand uwsu

Yours truly
CUMGruop (Project AMOGUS)

**Comment by MikoHeuken** [comment]
Hey AMOGUS Team,

you did a great job with the last blog, the only negative thing are the mockups. But I don´t think it´s that important and as long as the rest is fine it doesn´t disturb anyone. The blog is so well written and structured, that there aren´t any further questions from our team.

Best regards, Team UltimateRecipeHunter.

**Reply by jas20202** [reply]
Hello Team UltimateRecipeHunter,

thank you for your comment and feedback on our blog.
We already recieved a comment about the mockups and opened a task in Jira to make them prettier ^^
It's good to hear that our Blog posts are well written and structured and that no questions seem to be unanswered. If you do stumble across a question later on, feel free to contact us, we don't bite 😃 

Yours truly
CUMGruop (Project AMOGUS)

**Comment by Tinf21-B4-SE-Project** [comment]
Hey AMOGUS Team,

we like your Use-Case-Realization Specification. The only thing we would improve would be a uniform design for your UML sequence diagrams. If you want to create your diagram online, we would recommend the Draw lo website. On this there are prefabricated diagrams that you can easily customize.

Team cyBuy

**Reply by jas20202** [reply]
Hello there Team cyBuy,

nice to see you by. Thank you for your feedback. We are already fixing the sequence diagrams and we will open a Jira task for making them consistent. That means it will be done as soon as a team member is free ^^
Also thank you for your hint where we can realize the task, this helps a lot 😊 And we are glad that you like our UCRS.

Yours truly
CUMGruop (Project AMOGUS)
